recipes/proj/all/sqlite_init.py

```diff
@@ -0,0 +1,7 @@
+import sqlite3
```

Comment:
I think this is quite a hack as well, but I see we don't have the executable in the sqlite3 packages so it is the only way to go for now.

On the other hand, this might fail for Conan users that are using the installers (no Python installed).

My question is the following: If the sqlite3 library is expected to have the executable, why don't we package it? Any ideas @madebr @SSE4 @uilianries ?

Reply:
@danimtb I've submitted a PR to add sqlite3 CLI: https://github.com/conan-io/conan-center-index/pull/1004

Reply:
Please @btashton take a look at the sqlite3 approach and try to use the sqlite3 executable from the Conan package now that we got #1004 merged. Thanks!

Reply:
Looking at #1004, sqlite3 will fail when using as a build requirement and cross building (e.g. building an arm application on x86).